src/style.css

```diff
@@ -0,0 +1,133 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+ 
+.card__image {
+  margin: 32px auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+h1 {
+  margin: 0;
+}
+
+.card__title {
+  margin: 0 16px;
+  color: #060b35;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__product {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin: 4px 16px 16px;
+}
+
+.card__raiting {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 16px 24px;
+}
+
+p {
+  margin: 0;
+}
+
+.star {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  margin: 0 16px 24px;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat ;
+  width: 16px;
+  height: 16px;
+  margin: 0 2px;
+  background-position: center;
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.reviews {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.card__price {
+  margin: 0 16px 16px;
+  display: flex;
+  align-items: center;
+  /* flex-direction: row; */
+  justify-content: space-between;
+  font-weight: 400;
+}
+
+.card__value {
+  font-weight: 400;
+  font-size: 12px;
+  color: #616070;
+}
+
+.card__amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+a {
+  margin: auto;
+}
+
+.card__button {
+  text-align: center;
+  padding: 12px 0;
+  width: 166px;
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 700;
+  text-transform: uppercase;
+  background: #00acdc;
+  color: #fff;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+}
+
+.card__button:hover {
+  color: #00acdc;
+  text-align: center;
+  background: #fff;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
```

Comment:
```suggestion
.card__button:hover {
  color: #00acdc;
  background: #fff;
  border-color: #00acdc;
```
![Screenshot from 2023-05-17 13-08-32](https://github.com/mate-academy/layout_product-cards/assets/33029864/46d835bd-191c-4475-8a27-b797a42d5bf7)

make variable for colors which uses more than one time

Reply:
Ok